common-lib/test/cmr/common/test/util.clj

```diff
@@ -773,3 +773,15 @@
 
     "Set of numbers"
     #{1 7 52 3} 52))
+
+(deftest safe-lower-upper-case-test
+  (testing "safe-lowercase"
+    (is (= (util/safe-lowercase false) (str/lower-case false)))
+    (is (= (util/safe-lowercase true) (str/lower-case true)))
+    (is (= (util/safe-lowercase "StRing") (str/lower-case "StRing")))
+    (is (= (util/safe-lowercase nil) nil)))
+  (testing "safe-upperrcase"
```

Comment:
uppercase typo

Reply:
Sorry didn't see the comment. Will fix this in my next PR.